public/tracker/modules/task/directives/taskEditor/taskEditor.js

```diff
@@ -15,7 +15,13 @@ angular
                                   Team,
                                   toaster,
                                   TaskFile,
-                                  TagsList) {
+                                  TagsList,
+                                  $timeout) {
+
+                $scope.showElastic = false;
```

Comment:
Lets work with naming, who is Elastic? :) Ealastica/Elastigirl ? 
![Ealastica/Elastigirl ? ](http://vector-cartoons.com/images/incredibles11.gif)